docs/getting-started/security-ui.asciidoc

```diff
@@ -22,204 +23,69 @@ image::images/search-bar.png[]
 
-[float]
-[[rules-page]]
-=== Rules page
-
-The Rules page allows you to view and manage all detection rules. Refer to <<rules-ui-management, Manage detection rules>> for more information about prebuilt and custom rules.
+<<timelines-ui, *Timelines*>>: Investigate alerts and complex threats, such as lateral movement of malware across hosts in your network. Timelines are responsive and allow you to share your findings among other team members.
 
 [role="screenshot"]
-image::detections/images/all-rules.png[Shows the Rules page]
+image::images/timeline-ui.png[Shows the Timeline page]
 
-[float]
-[[exceptions-page]]
-=== Exception lists page
+TIP: Select the collapsable *Timeline* button at the bottom of the {security-app} to start an investigation.
 
-The Exception lists page allows you to view and manage all rule exceptions. Refer to <<detections-ui-exceptions, Rule exceptions and value lists>> for more information about rule exceptions.
+<<cases-overview, *Cases*>>: Open and track security issues.
 
 [role="screenshot"]
-image::detections/images/exceptions-page.png[Shows the Exception lists page]
-
-[float]
-[[hosts-ui]]
-=== Hosts page
+image::cases/images/cases-home-page.png[Shows the Cases page]
+*Explore*: Expand this section to view the following pages:
 
-The Hosts page provides key metrics for host-related security events and a set of interactive data tables. Refer to <<hosts-overview, Hosts page>> for more information.
+* <<hosts-overview, *Hosts*>>: View key metrics for host-related security events through graphs, charts, and interactive data tables.
 
 [role="screenshot"]
 image::management/hosts/images/hosts-ov-pg.png[Shows the Hosts page]
 
-
-[float]
-[[network-page]]
-=== Network page
-
-The Network page provides key network activity metrics via an interactive map and network event tables
-that enable interaction with Timeline. Refer to <<network-page-overview, Network page overview>> for more information.
+* <<network-page-overview, *Network*>>: View key network activity metrics through an interactive map and network event tables that interact with Timeline.
 
 [role="screenshot"]
 image::images/network-ui.png[Shows the Network page]
 
-[float]
-[[users-page-ov]]
-=== Users page
-
-The Users page provides a comprehensive overview of user data to help you understand authentication and user behavior within your environment. Refer to <<users-page, Users page>> for more information.
+* <<users-page, *Users*>>: Access a comprehensive overview of user data to help you understand authentication and user behavior within your environment.
 
 [role="screenshot"]
 image::images/users/users-page.png[Shows the Users page]
 
+*Get started page*: View a quick shortcut to add security integrations, which you can use to monitor your hosts and ingest data. Other {security-app} pages that show event data (for example, the Hosts and Network pages) display the get started prompt until event data has been ingested.
```

Comment:
```suggestion
*Get started page*: Quickly add security integrations that ingest data and monitor your hosts. Other {security-app} pages that show event data (for example, the Hosts and Network pages) provide the same option until event data has been ingested.
```
I don't feel like the second sentence is necessary, since I don't think users reading the first sentence will have any trouble finding a way to add integrations.

Reply:
+